<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Results</class>
 <widget class="QDialog" name="Results">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BMR BMI Results</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="3">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>208</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" rowspan="11">
    <widget class="QLabel" name="label_image">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Metabolism</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="line_meta">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>You should eat</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="line_sum">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="3">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>208</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Proteins</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" rowspan="2">
    <widget class="QLineEdit" name="line_protein">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3" rowspan="2">
    <widget class="QLineEdit" name="line_protein2"/>
   </item>
   <item row="6" column="0" rowspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Fats</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="line_fat">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLineEdit" name="line_fat2"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Carbohydrates</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="line_carbs">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="line_carbs2"/>
   </item>
   <item row="9" column="1" colspan="3">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>208</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1" rowspan="2" colspan="3">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>208</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="4" rowspan="6">
    <widget class="QTableWidget" name="table_bmi">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>BMI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>&lt;18.5</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Underweight</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>18.5-25</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>25-30</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>Overweight</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>30-35</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>Obese</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>&gt;35</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string>Extremely obese</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>BMI</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="line_bmi">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>79</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1" colspan="3">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>208</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Goal achieved</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QDateEdit" name="date_goal">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="3">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>208</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QPushButton" name="button_save">
     <property name="text">
      <string>Save results</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_meta</tabstop>
  <tabstop>line_sum</tabstop>
  <tabstop>line_protein</tabstop>
  <tabstop>line_protein2</tabstop>
  <tabstop>line_fat</tabstop>
  <tabstop>line_fat2</tabstop>
  <tabstop>line_carbs</tabstop>
  <tabstop>line_carbs2</tabstop>
  <tabstop>line_bmi</tabstop>
  <tabstop>date_goal</tabstop>
  <tabstop>button_save</tabstop>
  <tabstop>table_bmi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
